esmero-php-fpm/Dockerfile.dev

```diff
@@ -0,0 +1,143 @@
+# build with:
+# sudo docker build -f Dockerfile.dev --build-arg LOCAL_IP=<your machine IP> -t esmero-php:development . --no-cache`
+# from https://www.drupal.org/docs/8/system-requirements/drupal-8-php-requirements
+FROM php:7.3-fpm-alpine
+
+# should be passed from the build command
+ARG LOCAL_IP
+
+# install the PHP extensions we need
+# postgresql-dev is needed for https://bugs.alpinelinux.org/issues/3642
+RUN set -eux; \
+    \
+	apk add --no-cache --virtual .build-deps \
+		coreutils \
+    		zlib-dev \
+		libxml2-dev \
+		libxslt-dev \
+		freetype-dev \
+		libjpeg-turbo-dev \
+		libpng-dev \
+		libzip-dev \
+		postgresql-dev \
+		autoconf \
+        g++ \
+        make \
+	; \
+	\
+	docker-php-ext-configure gd \
+		--with-freetype-dir=/usr/include \
+		--with-jpeg-dir=/usr/include \
+		--with-png-dir=/usr/include \
+	; \
+	\
+	docker-php-ext-install -j "$(nproc)" \
+		gd \
+		zip \
+		pdo_mysql \
+		mysqli \
+		xsl \
+		opcache \
+		exif \
+		pgsql \
+		pdo_pgsql \
+		bcmath \
+	; \
+	\
+	# install x-debug
+	pecl install xdebug-2.8.0; \
+	runDeps="$( \
+		scanelf --needed --nobanner --format '%n#p' --recursive /usr/local \
+			| tr ',' '\n' \
+			| sort -u \
+			| awk 'system("[ -e /usr/local/lib/" $1 " ]") == 0 { next } { print "so:" $1 }' \
+	)"; \
+	apk add --virtual .drupal-phpexts-rundeps $runDeps; \
+	apk del .build-deps
+	
+# set recommended PHP.ini settings
+
+RUN { \
```

Comment:
So not sure about this. First, if you are really aiming at a different container, why extend base php:7.3-fpm-alpine and not our existing esmero-php one which has already a lot of this deployed?
That would leave this one basically with the xdebug installation via pecl install xdebug-2.8.0; 
Also, could we not simply mount/enable the /usr/local/etc/php/conf.d/xdebug.ini during runtime instead of building it during creation? That would allow people to change settings without having to setup anything?

Reply:
Ok, this is resolved. Good!